src/flows/pipes/Table/index.ts

```diff
@@ -1,5 +1,5 @@
 import View from './view'
-import {parse} from '@influxdata/flux'
+import {parseQuery} from 'src/shared/contexts/query'
```

Comment:
why?

Reply:
good catch. needs to be wrapped in a flag